packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -183,7 +196,8 @@ export class Bytecblock extends Op {
   readonly interpreter: Interpreter;
 
   /**
-   * @param args words list extracted from line: {byte block}
+   * Description: Store blocks of bytes in bytecblock
+   * @param args Expected arguments: [byte block]
```

Comment:
What does `[byte block]` mean? Expected arguments is an array of bytes. (eg. `bytecblock "a" "b" "c"`)

Reply:
hmm, just checked TEAL opcode documentation `intcblock` and `bytecblock` are wrongly implementated.

Reply:
** Discussion 
- On call discussed and checked both the opcodes, docs mentioned that we require a length for both, but when running teal codes, that's not the case.